tests/unit/cmd/test_ignore.py

```diff
@@ -212,26 +213,14 @@ def test_do_not_duplicate_last_found_secrets(client, isolated_fs):
     )
     cache = Cache()
 
-    cache.add_found_policy_break(policy_break, "a")
-    cache.add_found_policy_break(policy_break, "b")
-
-    assert len(cache.last_found_secrets) == 1
-
-
-def test_do_not_add_policy_breaks_to_last_found(client, isolated_fs):
```

Comment:
Not clear in the diff: I removed `test_do_not_add_policy_breaks_to_last_found`, which was not useful anymore.

What was tested was that non-secret policy breaks were not added to the cache (GIVEN/WHEN/THEN was not accurate).